kglib/kgcn/pipeline/pipeline.py

```diff
@@ -82,18 +86,28 @@ def pipeline(graphs,
                 node_output_size=node_output_size)
 
     learner = KGCNLearner(kgcn,
-                          num_processing_steps_tr=num_processing_steps_tr,
-                          num_processing_steps_ge=num_processing_steps_ge)
-
-    train_values, test_values, tr_info = learner(tr_input_graphs,
+                          num_processing_steps_tr=num_processing_steps_tr, # These processing steps indicate how many message-passing iterations to do for every training / testing step
+                          num_processing_steps_ge=num_processing_steps_ge,
+                          save_fle=f'{output_dir}/{save_fle}',
+                          reload_fle=f'{output_dir}/{reload_fle}')
+
+    # only test
+    if not Path(output_dir / reload_fle).is_dir() and do_test == True:
```

Comment:
This line currently causes a test to fail with the following error:
```
======================================================================
ERROR: test_learning_is_done (__main__.TestDiagnosisExample)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/circleci/.cache/bazel/_bazel_circleci/046479231ae58926fd02b6cd39e690ed/execroot/kglib/bazel-out/k8-fastbuild/bin/tests/end_to_end/diagnosis.runfiles/kglib/tests/end_to_end/kgcn/diagnosis.py", line 39, in test_learning_is_done
    solveds_tr, solveds_ge = diagnosis_example()
  File "/home/circleci/.cache/bazel/_bazel_circleci/046479231ae58926fd02b6cd39e690ed/execroot/kglib/bazel-out/k8-fastbuild/bin/tests/end_to_end/diagnosis.runfiles/kglib/kglib/kgcn/examples/diagnosis/diagnosis.py", line 108, in diagnosis_example
    output_dir=f"./events/{time.time()}/")
  File "/home/circleci/.cache/bazel/_bazel_circleci/046479231ae58926fd02b6cd39e690ed/execroot/kglib/bazel-out/k8-fastbuild/bin/tests/end_to_end/diagnosis.runfiles/kglib/kglib/kgcn/pipeline/pipeline.py", line 95, in pipeline
    if not Path(output_dir / reload_fle).is_dir() and do_test == True:
TypeError: unsupported operand type(s) for /: 'str' and 'str'

----------------------------------------------------------------------
Ran 1 test in 255.269s

FAILED (errors=1)
```

You can see the test [here](https://app.circleci.com/pipelines/github/graknlabs/kglib/260/workflows/7c6d87fa-d16a-409f-80ad-152e59811a67/jobs/1084). You can navigate to the tests via the tick or cross that's visible on the right of each commit on the PR. You can also run all of the tests locally on your own machine if you're up and running with bazel.

This line looks like it should be something like `Path(f'{output_dir}/{reload_fle}')`

Reply:
Error was caused because we use pathlib for paths. 
I think I fixed it. Will push it soon.